grpc/src/compiler/go_generator.cc

```diff
@@ -70,12 +70,11 @@ void GenerateImports(grpc_generator::File *file, grpc_generator::Printer *printe
 	printer->Print("//If you make any local changes, they will be lost\n");
```

Comment:
Sorry. my comment was not about this line, it is about the whole file. I'm saying it is better to change go_generator.cc in upstream project.